src/lib/components/forms/InstructorFeedbackForm.svelte

```diff
@@ -105,7 +118,12 @@
         ).catch((error) => {
           console.log(error)
         })
+        updateDoc(
+          doc(db, 'classesSpring24', frozenUser.object.uid),
+          { feedbackCompleted: feedbackCompleted },
+        )
         alert.trigger('success', 'Class Feedback saved!')
+        setTimeout(() => location.reload(), 1000)
```

Comment:
why is reload needed

Reply:
the changes for the class's status color rendering don't propagate until the page reloads -- maybe not necessary though